src/routes/process-user-vocabulary/UserVocabularyProcessScreen.tsx

```diff
@@ -94,8 +94,8 @@ const UserVocabularyProcessScreen = ({ navigation }: UserVocabularyProcessScreen
 
       const imagePaths = await Promise.all(
         images.map(async (image, index) => {
-          const path = `${DocumentDirectoryPath}/image-${id}-${index}.txt`
-          await writeFile(path, image, 'utf8')
+          const path = `file:///${DocumentDirectoryPath}/image-${id}-${index}.jpg`
+          await moveFile(image, path)
```

Comment:
File should not be moved, as the file should only be cached, when the the camera overlay "saves" = caches it. So if a user cancels the creation of a vocabulary no image files that do not belong to a saved images are there. Only if a user vocabulary is saved, the images should be saved.

Reply:
the file will be cached and only moved if you save the vocabulary.